iOS/FVM/FVM/FVMCarModelViewController.swift

```diff
@@ -33,10 +34,15 @@ public class FVMCarModelViewController : SCNView {
     private func setupGestures() {
         let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleTapGesture(_:)))
         let pinchGesture = UIPinchGestureRecognizer(target: self, action: #selector(handlePinchGesture(_:)))
+        let panGesture = UIPanGestureRecognizer(target: self, action: #selector(handlePanGesture(_:)))
+        panGesture.minimumNumberOfTouches = 1
+        panGesture.maximumNumberOfTouches = 2
```

Comment:
Shouldn't it be part of the PanGestureHandler.swift? As a method returning UIPanGestureRecognizer, for example